ggshield/cmd/iac/scan/iac_scan_common_options.py

```diff
@@ -95,3 +98,63 @@ def update_context(
 
     if minimum_severity is not None:
         config.user_config.iac.minimum_severity = minimum_severity
+
+
+def augment_unignored_issues(
+    ctx: click.Context,
+    result: Union[IaCScanResult, IaCDiffScanResult, IaCSkipScanResult, None],
+) -> None:
+    """
+    GIVEN a list of vulnerabilities from a scan result
+    WHEN ignored policies and paths are configured but outdated
+    THEN augment the vulnerability with the date it was last ignored
+    """
+    if isinstance(result, IaCScanResult):
+        incidents_list = result.entities_with_incidents
+    elif isinstance(result, IaCDiffScanResult):
+        incidents_list = result.entities_with_incidents.new
```

Comment:
We want the text only for new incident ?

What about unchanged ?

Reply:
If I'm not mistaken, incidents details is only shown for new incidents and this is where the message about the date is shown. Or is there a verbose option that enables to show details for all incidents (new, unchanged and deleted)? Or do we want this augmentation for all incidents anyway, just so we keep them consistent?

Reply:
When verbose is on, we also display unchanged incidents.
We might want to validate with the product if we want to display the end of grace period in this case or not

Reply:
Validated decision: we augment them all.